canvas_modules/common-canvas/src/common-properties/controls/checkbox/checkbox.scss

```diff
@@ -26,9 +26,7 @@
 	}
 	.bx--form-item.bx--checkbox-wrapper {
 		flex: initial; // override carbon style to show tooltipIcon next to checkbox label
-	}
-	.properties-validation-message {
-		flex: 0 1 100%;
+		margin-block-start: 0.1875rem;
```

Comment:
Where does the 0.1875 come from?

Reply:
Thanks for catching that. I added that CSS on `main` branch because carbon 11 adds `margin-block-start: 0` which reduced the spacing between 2 checkboxes. 

But that CSS isn't needed in v12.x branch. Updated PR.